### PR TITLE
updating tomcat jdbc version

### PR DESCRIPTION
**Pull request by srallapa**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@
     <jcifs.version>1.3.3</jcifs.version>
     <woodstox-core-asl.version>4.4.1</woodstox-core-asl.version>
     <xmlunit.version>1.5</xmlunit.version>
+    <tomcat-version>10.1.24</tomcat-version>
```
Comment (lucboudreau) [inline core/pom.xml:36]: Can this be sync'd with the parent pom instead?

Comment (ddiroma) [reply core/pom.xml:36]: @srallapa Also, can this property more specific to jdbc instead of <tomcat.version>?

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

File: core/pom.xml (hunk 1 above)
Comment (srallapa) [reply core/pom.xml]: > Can this be sync'd with the parent pom instead?

Yes I have updated from the parent pom

**Review by ddiroma — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Pull request merged**